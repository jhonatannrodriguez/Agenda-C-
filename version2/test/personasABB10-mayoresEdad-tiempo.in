crearArbolPersonas
mayoresEdadArbolPersonasTiempo 10000 2 19900
liberarArbolPersonas

Fin
