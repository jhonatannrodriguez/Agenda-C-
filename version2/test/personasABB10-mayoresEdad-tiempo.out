1>2>Persona 9951: Juan, 19902 años
Persona 9952: Juan, 19904 años
Persona 9953: Juan, 19906 años
Persona 9954: Juan, 19908 años
Persona 9955: Juan, 19910 años
Persona 9956: Juan, 19912 años
Persona 9957: Juan, 19914 años
Persona 9958: Juan, 19916 años
Persona 9959: Juan, 19918 años
Persona 9960: Juan, 19920 años
Persona 9961: Juan, 19922 años
Persona 9962: Juan, 19924 años
Persona 9963: Juan, 19926 años
Persona 9964: Juan, 19928 años
Persona 9965: Juan, 19930 años
Persona 9966: Juan, 19932 años
Persona 9967: Juan, 19934 años
Persona 9968: Juan, 19936 años
Persona 9969: Juan, 19938 años
Persona 9970: Juan, 19940 años
Persona 9971: Juan, 19942 años
Persona 9972: Juan, 19944 años
Persona 9973: Juan, 19946 años
Persona 9974: Juan, 19948 años
Persona 9975: Juan, 19950 años
Persona 9976: Juan, 19952 años
Persona 9977: Juan, 19954 años
Persona 9978: Juan, 19956 años
Persona 9979: Juan, 19958 años
Persona 9980: Juan, 19960 años
Persona 9981: Juan, 19962 años
Persona 9982: Juan, 19964 años
Persona 9983: Juan, 19966 años
Persona 9984: Juan, 19968 años
Persona 9985: Juan, 19970 años
Persona 9986: Juan, 19972 años
Persona 9987: Juan, 19974 años
Persona 9988: Juan, 19976 años
Persona 9989: Juan, 19978 años
Persona 9990: Juan, 19980 años
Persona 9991: Juan, 19982 años
Persona 9992: Juan, 19984 años
Persona 9993: Juan, 19986 años
Persona 9994: Juan, 19988 años
Persona 9995: Juan, 19990 años
Persona 9996: Juan, 19992 años
Persona 9997: Juan, 19994 años
Persona 9998: Juan, 19996 años
Persona 9999: Juan, 19998 años
Bien.
3>4>Fin.
